_includes/card.html

```diff
@@ -0,0 +1,27 @@
+<!--
+Variables:
+
+title
+image1110
+image690
+image375
+imagealt
+overlayheadline
+text
+buttonlink
+buttontext
+-->
+
+<section id="card">
+  <h2>{{include.title}}</h2>
+  <div class="card rounded shadow" style="height: 400px;">
+    <img srcset="{{include.image375}} 375w,
+                 {{include.image690}} 690w,
+                {{include.image1110}} 1110w" sizes="100vw" class="card-img" style="height: 400px; object-fit:cover; " alt="{{include.imagealt}}">
+    <div class="card-img-overlay d-flex flex-column">
+      <h1 class="text-white pl-3">{{include.overlayheadline}}</h1>
```

Comment:
Maybe we could add some dark drop shadow here to improve readability?

![grafik](https://user-images.githubusercontent.com/2820734/80893642-8c1eb700-8cd4-11ea-96c7-50b6f6c36cff.png)


```css
text-shadow: 0 0 5pt rgba(0, 0, 0, 50%);
```

Reply:
Top,

wollte hier in einem späteren Commit noch etwas Text zum TechAachen hinzufügen.